### PR TITLE
doc(taxes): add taxes info in other section

### PR DESCRIPTION
**Pull request by michaelponrajah**
## Description

Add taxes in other section of the guide

### CODE REVIEW

**Review by ansmonjol — APPROVED** [review]

File: guide/invoicing/download-invoices.mdx
Comment (sarkissianraffi) [inline guide/invoicing/download-invoices.mdx:1]: Is this an empty string? @michaelponrajah

File: api-reference/taxes/create.mdx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+```python Python
+from lago_python_client.client import Client
+from lago_python_client.exceptions import LagoApiError
+from lago_python_client.models import Tax
+
+client = Client(api_key='__YOUR_API_KEY__')
+
+tax = Tax(
+		name="TVA",
```
Comment (sarkissianraffi) [inline api-reference/taxes/create.mdx:34]: extra space here @michaelponrajah

File: api-reference/taxes/create.mdx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    repair_broken_state(e)
+```
+
+```ruby Ruby
+require 'lago-ruby-client'
+
+client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+
+tax = {
+	name: "TVA",
```
Comment (sarkissianraffi) [inline api-reference/taxes/create.mdx:53]: Same extra space @michaelponrajah

File: api-reference/taxes/object.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      Name of the tax.
+    </ResponseField>
+    <ResponseField name="code" type="string" required>
+      Unique code used to identify the tax associated with the API
+      request.
+    </ResponseField>
+    <ResponseField name="rate" type="float" required>
+      The percentage rate of the tax
+    </ResponseField>
+    <ResponseField name="description" type="string | null">
```
Comment (sarkissianraffi) [inline api-reference/taxes/object.mdx:40]: same

File: api-reference/taxes/object.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      Internal description of the tax.
+    </ResponseField>
+    <ResponseField name="applied_to_organization" type="boolean required">
```
Comment (sarkissianraffi) [inline api-reference/taxes/object.mdx:43]: bad closing for type @michaelponrajah

File: api-reference/taxes/object.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+title: "The billable metric object"
```
Comment (sarkissianraffi) [inline api-reference/taxes/object.mdx:2]: Bad title @michaelponrajah

**Review by sarkissianraffi — CHANGES_REQUESTED** [review]

**Pull request merged**